### ISSUE TITLE
Is this project abandonded? 

### ISSUE THREAD

**Issue by Nicedude-git** [pump-io/pump.io#1715]
Is this project abandonded? As in what concerns someone who wants to setup a new instance?

**Comment by vxcamiloxv** [comment]
No, you can see the commits in master branch and teh last was a couple of days/hours, the 6.0.0 is ready to be released cc: @strugee

**Comment by Neustradamus** [comment]
In 2023, badly, I think, yes...

**Comment by rakotomandimby** [comment]
Posting from 2024, yes it is unfortunately dead...

**Comment by vasyugan** [comment]
In 2026, I have to remark that who is dead ususally stays dead, resurrections are rare... Interesting that identi.ca is still around (although it is down frequently).

**Comment by Neustradamus** [comment]
There are vulnerabilities:
- https://github.com/pump-io/pump.io/issues/1730

I have requested the details.
